app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,23 @@
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+.footer {
+   position: fixed;
+   left: 0;
+   bottom: 0;
+   width: 100%;
+   height: 10%;
+   background-color: rgb(150, 150, 0);
+   background-image:url("https://static.vecteezy.com/system/resources/previews/000/663/569/original/vector-ping-pong-table-tennis.png");
+   background-position: center;
+   color: white;
+   text-align: center;
+}
```

Comment:
You can make it more civilized by moving all css stuff to its place - in our case that is `app/assets/stylesheets/application.css`